methods_test.test.js

```diff
@@ -0,0 +1,42 @@
+// import capitalize from './methods_to_test';
+const { capitalize, reverseString, analyse, Calculator, Cypher } = require('./methods_to_test.js')
+
+
+test("capitalize", () => {
```

Comment:
## CRUCIAL

Let us write at least 3 test cases (considering edge cases) for each function. We cannot rely that our functions/methods are correct just because it passes one test.